src/models/Offer.js

```diff
@@ -110,9 +110,28 @@ OfferSchema.query.current = function() {
     });
 };
 
+OfferSchema.query.activeOffersCount = function(canditateCompany) {
+    return this.where({
+        publishDate: {
+            $lte: new Date(Date.now()),
+        },
+        publishEndDate: {
+            $gt: new Date(Date.now()),
+        },
+        owner: {
+            $eq: canditateCompany.owner,
+        },
+    });
+};
+
 const Offer = mongoose.model("Offer", OfferSchema);
+const MAX_ACTIVE_OFFERS_ALLOWED = 10;
+
 
 // Useful for testing correct field implementation
 // console.log("DBG: ", OfferSchema.path("location"));
 
-module.exports = Offer;
+module.exports = {
+    Offer,
+    MAX_ACTIVE_OFFERS_ALLOWED,
+};
```

Comment:
This is probably a cause of problems.
Here, you are saying that you want to export an object which has two properties: `Offer`, and `MAX_ACTIVE_OFFERS_ALLOWED`. By not specifying the values for those keys, they will get the value of the variables with the same name, by default, which is ok. However, when you import it, you won't be importing the Offer model anymore, you will be importing an object which Offer key will be the Offer model, therefore you would actually need to do something like `const Offer = require("./Offer").Offer;` to import it, which is not that good.

What you can do instead is do it like this:
```js
module.exports = Offer;
module.exports.MAX_ACTIVE_OFFERS_ALLOWED = MAX_ACTIVE_OFFERS_ALLOWED;
```

With this, you can import the Offer model with `const Offer = require("./Offer");` and MAX_ACTIVE_OFFERS_ALLOWED with `const MAX_ACTIVE_OFFERS_ALLOWED = require("./Offer").MAX_ACTIVE_OFFERS_ALLOWED;`

**This was an explanation**, however, what I want you to do is use the constants file, in models/constants/Offer.js and create the maxActiveOffersAllowed rule there. This way, you only export the Offer model from this file